collection.php

```diff
@@ -1,15 +1,9 @@
+<!DOCTYPE html>
 <?php
 require_once '.' . DIRECTORY_SEPARATOR . 'displayDBContentFunction.php';
-
-$db = new PDO ('mysql:host=db; dbname=OfficeQuotesDB', 'root', 'password');
-$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-$query = $db->prepare('SELECT * FROM `quotes` INNER JOIN `images` ON `quotes`.`id` = `images`.`quotes-ID`');
-$query->execute();
-$quotes = $query->fetchAll();
+require '.' . DIRECTORY_SEPARATOR . 'collectionDBConnection.php';
```

Comment:
Why not require once as well?

You'd only ever want to have one active connection